Current "main" fails to gen_docs

**Describe the bug**
```text
.  .  .
Generate XML output for dir /Users/ur20980/src/liboqs/src/common/aes/
Generate XML output for dir /Users/ur20980/src/liboqs/src/common/
Generate XML output for dir /Users/ur20980/src/liboqs/src/kem/
Generate XML output for dir /Users/ur20980/src/liboqs/src/common/rand/
Generate XML output for dir /Users/ur20980/src/liboqs/src/common/sha2/
Generate XML output for dir /Users/ur20980/src/liboqs/src/common/sha3/
Generate XML output for dir /Users/ur20980/src/liboqs/src/sig/
Generate XML output for dir /Users/ur20980/src/liboqs/src/
Generating XML output for the main page
/Users/ur20980/src/liboqs/README.md:49: error: unable to resolve reference to 'oqs_algs_enabled' for \ref command
/Users/ur20980/src/liboqs/README.md:126: error: unable to resolve reference to 'build_shared_libs' for \ref command
Running plantuml with JAVA...
type lookup cache used 394/65536 hits=1015 misses=394
symbol lookup cache used 433/65536 hits=3672 misses=433
finished...
Exiting...
make[3]: *** [CMakeFiles/gen_docs] Error 1
make[2]: *** [CMakeFiles/gen_docs.dir/all] Error 2
make[1]: *** [CMakeFiles/gen_docs.dir/rule] Error 2
make: *** [gen_docs] Error 2
```

**To Reproduce**
Clone, configure with CMake, do `make`, then `make gen_docs`. Observe the above error.

**Expected behavior**
Successful generation of documentation. _Speaking of which, it would be awfully nice to be able to output docs in PDF format._

**Environment (please complete the following information):**
 - OS: MacOS Sonoma 14.6.1
 - OpenSSL version 3.3.1 4 Jun 2024 (Library: OpenSSL 3.3.1 4 Jun 2024)
 - Compiler version used: Apple clang version 15.0.0 (clang-1500.3.9.4)
 - Build variables used: `-DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX=/opt/local`
 - liboqs version: `main` branch

--- a/README.md
+++ b/README.md
@@ -44,7 +44,7 @@ The only algorithms in `liboqs` that implement NIST standards are the [`ML-KEM`]
 
 Falcon and SPHINCS+ have also been [selected for standardization](https://csrc.nist.gov/Projects/post-quantum-cryptography/selected-algorithms-2022), but the `liboqs` implementations of these algorithms are currently tracking Round 3 submissions and not NIST standards drafts.
 
-All names other than `ML-KEM` and `ML-DSA` are subject to change. `liboqs` makes available a [selection mechanism for algorithms on the NIST standards track, continued NIST competition, or purely experimental nature by way of the configuration variable OQS_ALGS_ENABLED](CONFIGURE.md#oqs_algs_enabled). By default `liboqs` is built supporting all, incl. experimental, PQ algorithms listed below.
+All names other than `ML-KEM` and `ML-DSA` are subject to change. `liboqs` makes available a [selection mechanism for algorithms on the NIST standards track, continued NIST competition, or purely experimental nature by way of the configuration variable OQS_ALGS_ENABLED](CONFIGURE.md#oQS_ALGS_ENABLED). By default `liboqs` is built supporting all, incl. experimental, PQ algorithms listed below.
 
 #### Key encapsulation mechanisms
 
@@ -122,7 +122,7 @@ Various `cmake` build options to customize the resultant artifacts are available
 
 The following instructions assume we are in `build`.
 
-3. By default the main build result is `lib/liboqs.a`, a static library. If you want to build a shared/dynamic library, append [`-DBUILD_SHARED_LIBS=ON`](CONFIGURE.md#build_shared_libs) to the `cmake -GNinja ..` command above and the result will be `lib/liboqs.so|dylib|dll`. The public headers are located in the `include` directory. There are also a variety of programs built under the `tests` directory:
+3. By default the main build result is `lib/liboqs.a`, a static library. If you want to build a shared/dynamic library, append [`-DBUILD_SHARED_LIBS=ON`](CONFIGURE.md#bUILD_SHARED_LIBS) to the `cmake -GNinja ..` command above and the result will be `lib/liboqs.so|dylib|dll`. The public headers are located in the `include` directory. There are also a variety of programs built under the `tests` directory:
 
 	- `test_kem`: Simple test harness for key encapsulation mechanisms
 	- `test_sig`: Simple test harness for signature schemes

